src/screen.c

```diff
@@ -3,8 +3,6 @@
 #include "io.h"
 #include "serial_port.h"
 
-static u16 *display_buffer = (u16 *)0xB8000;
```

Comment:
`display_buffer` as a concept shouldn't be removed. The `where` thing doesn't really make sense and is not descriptive at all.